### ISSUE TITLE
[Feature] Documentation request

### ISSUE THREAD

**Issue by NathanC** [end-4/dots-hyprland#571]
**What would you like to be added?**

An overview of software installed/reasons they were chosen. Basically a high level table breakdown of the core decisions made here and packages used.

**How will it help**

I have no context for a lot of this. I've used Arch extensively, but never tiling windows managers and never waylands. I just went through the hyprland setup and got the baseline working and set up waybar, but for this install setup I can't even tell what status bar is being used because I don't even know the names of any of them.

Basically, if you want this to be useful to newcomers to hyprland, bridging the context gap by an overview of high level decisions made and core software (what status bar, what notification daemon, QT vs GTK etc) would be helpful.

I'm just looking for some more documentation to counter the [curse of knowledge](https://en.wikipedia.org/wiki/Curse_of_knowledge). You have a wiki so seems like it's something in line with what you're trying to do here, and this would have been really helpful to me. Thanks for maintaining this!

**Comment by end-4** [comment]
I honestly totally forget what software is installed, except the ones I mainly work with like 
- Hyprland (the window manager (more technically-accurate term is compositor))
- AGS (a GTK widget system, responsible for the bar)

If you're going to ask me to tell you what a random package on your system is for, I'll probably shrug

**Comment by end-4** [comment]
ughhhhh
So do you want me to make an overview of stuff that were installed by the dotfiles install script or do you want a general utility that explains everything on your system?

**Comment by NathanC** [comment]
lol no, not looking for something to explain everything on my system. 

> AGS (a GTK widget system, responsible for the bar) 

Would have been very useful in the wiki or top level README, along with fuzzle etc, just a brief section on core components/first layer of the dependency tree. By the time you responded, I figured that's what was creating the bar, but it wasn't at all clear at first. Looking for reasonable *entry points* into Googling, that's all.

With the documentation you have and the stated focus on having an accessible system, figured I'd be worth saying what I wished was there as a top-level overview. If you're cool with it, I can add a brief section in the README as I'm digging through this, even just a little bullet point list with stuff like "AGS (a GTK widget system, responsible for the bar) " would be helpful to people.
